Update ReadMe (#15)
ferryhmm/oii: README.md

# osu!ii - Improvement Indicator

A browser extention that adds the improvement indicator (ii) to osu! profile. The ii is a metric that compares your pp to the average player.

## General

> The ii value is calculated by taking the average ii value of all players with the same playtime amount and multiplying it by your playtime amount. The formula for calculating the ii value is as follows:

Reviewer: ```suggestion
The ii value is calculated by taking the average playtime of all players for a given pp amount and dividing it by your playtime amount. The formula for calculating the ii value is as follows:
```
